.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,113 @@
+      multiple: false
+      options:
+        - "Managed Services (Astronomer, Composer, MWAA etc.)"
+        - "Official Apache Airflow Helm Chart"
+        - "Other 3rd-party Helm chart"
+        - "Docker-Compose"
+        - "Other Docker-based deployment"
+        - "Virtualenv installation"
+        - "Other"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Deployment details
+      description: Additional description of your deployment.
+      placeholder: >
+        Enter any relevant details of your deployment. Especially version of your tools,
+        software (docker-compose, helm, k8s, etc.), any customisation and configuration you added
+  - type: textarea
+    attributes:
+      label: What happened
+      description: Describe what happened.
+      placeholder: >
+        Please provide a context in which the problem occured and explain what happened!
+  - type: textarea
+    attributes:
+      label: What you expected to happen
+      description: What do you think went wrong?
+      placeholder: >
+        Please explain why do you think the behaviour is erronous. It is extremely helpful if you copy&paste
+        the fragment of logs showing the exact error messages or wrong behaviour and screenshots for the
+        UI problems or youtube link to video. You can include images/screen-casts etc. by drag-dropping the
+        image here
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: >
+        What should we do to reproduce the problem? If you are not able to provide a reproducible case,
+        please open [Discussion](https://github.com/apache/airflow/discussions) instead!
+      placeholder: >
+        Please makes sure you provide a reproducible step-by-step case of how to reproduce the problem.
+        As minimally and precisely as possible. Keep in mind we do not have access to your cluster or dags.
+        Rember that non-reproducible issues will be closed! Often opening a discussion is better idea!
+  - type: textarea
+    attributes:
+      label: Anything else
+      description: Anything else we need to know?
+      placeholder: >
+        How often does this problem occur? Once? Every time etc?
+        Any relevant logs to include? Put them here inside fenced
+        ``` ``` blocks or inside a foldable details tag if it's long:
+        <details><summary>x.log</summary> lots of stuff </details>
+  - type: checkboxes
+    attributes:
+      label: Are you willing to submit PR?
+      description: >
+        This is absolutely not required, but we are happy to guide you in contribution process
+        especially if you already have a good understanding of how to implement the fix.
+        Airflow is a community-managed project and we love to bring new contributors in.
+        Find us in #airflow-how-to-pr on Slack!
+      options:
+        - label: Yes I am willing to submit PR providing that I get guidance.
```

```suggestion
        - label: Yes I am willing to submit a PR!
```

I like this better, but 🤷‍♂️